upphovsrätt.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<!-- saved from url=(0069)https://ntisod.github.io/wutv1test-Klivaren0mar/upphovsr%C3%A4tt.html -->
+<html lang="sv"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./upphovsrätt_files/styles.css">
```

Comment:
Se till att skriva en titel som är mer beskrivande för sidans innehåll.
```suggestion
    <title>Upphovsrätt</title>
    <link rel="stylesheet" href="./upphovsrätt_files/styles.css">
```